README.md.template

```diff
@@ -22,18 +22,5 @@ under the terms specified in the [LICENSE] file.
 
 [LICENSE]: /LICENSE
 
-## About
-
-$(PROJECT_NAME) is maintained by $(MAINTAINERS).
```

Comment:
We'll lose these variables for now. I updated lots of repos and didn't see this line being used here. The repos already had their maintainers names alongside the copyright section, so it won't be a huge loss. But it's always something we can add back later.